server/config/index.js

```diff
@@ -102,10 +102,13 @@ if (process.env.SCHEMA_LISTENER_CONFIG) {
 
 if (process.env.KEYCLOAK_CONFIG_FILE) {
   try {
+    const keycloakHost = process.env.KEYCLOAK_HOST || 'localhost'
+    const keycloakPort = process.env.KEYCLOAK_PORT || '8080'
     const keycloakConfig = fs.readFileSync(process.env.KEYCLOAK_CONFIG_FILE, 'utf-8')
 
     config.securityServiceConfig.type = 'keycloak'
     config.securityServiceConfig.config = JSON.parse(keycloakConfig)
+    config.securityServiceConfig.config['auth-server-url'] = `http://${keycloakHost}:${keycloakPort}/auth`
```

Comment:
Just a minor thing. This is a bit of a hack and would break existing functionality, requiring changes in the data-sync-apb. We should write/store a keycloak config file with the correct values inside the `integration_tests` folder and provide that to the server using `process.env.KEYCLOAK_CONFIG_FILE`

Reply:
Cool. I have it in my todo list and was just wondering how to achieve this. Rewriting the keycloak config file is good idea.